torch/_torch_docs.py

```diff
@@ -3561,6 +3561,38 @@ def merge_dicts(*dicts):
     RuntimeError: bool value of Tensor with no values is ambiguous
 """.format(**common_args))
 
+add_docstr(torch.kron,
+           r"""
+kron(input, other, *, out=None) -> Tensor
+
+Computes the Kronecker product of :attr:`input` and :attr:`other`.
+
+If :attr:`input` is a :math:`(n \times m)` tensor, :attr:`other` is a
+:math:`(k \times l)` tensor, the result will be a :math:`(n*k \times m*l)` tensor.
+
+Supports real and complex inputs.
+
+Arguments:
+    input (Tensor): the first tensor to be multiplied
```

Comment:
They're not really "multiplied." This can just leave things as input (Tensor) and other (Tensor) without elaboration.